packages/devtools-reps/src/object-inspector/tests/object-inspector.js

```diff
@@ -816,3 +816,89 @@ describe("getChildren", () => {
     expect(nodePaths).toEqual(childrenPaths);
   });
 });
+
+describe("getValue", () => {
+  it("get the value from contents.value", () => {
+    let item = {
+      contents: {
+        value: "my value"
+      }
+    };
+    expect(getValue(item)).toBe("my value");
+
+    item = {
+      contents: {
+        value: 0
+      }
+    };
+    expect(getValue(item)).toBe(0);
+
+    item = {
+      contents: {
+        value: false
+      }
+    };
+    expect(getValue(item)).toBe(false);
+
+    item = {
+      contents: {
+        value: null
+      }
+    };
+    expect(getValue(item)).toBe(null);
+  });
+
+  it("get the value from contents.getterValue", () => {
+    let item = {
+      contents: {
+        getterValue: "my getter value"
+      }
+    };
+    expect(getValue(item)).toBe("my getter value");
+
+    item = {
+      contents: {
+        getterValue: 0
+      }
+    };
+    expect(getValue(item)).toBe(0);
+
+    item = {
+      contents: {
+        getterValue: false
+      }
+    };
+    expect(getValue(item)).toBe(false);
+
+    item = {
+      contents: {
+        getterValue: null
+      }
+    };
+    expect(getValue(item)).toBe(null);
+  });
+
+  it("get the value from getter and setter", () => {
+    let item = {
+      contents: {
+        get: "get"
+      }
+    };
+    expect(getValue(item)).toEqual({get: "get"});
+
+    item = {
+      contents: {
+        set: "set"
+      }
+    };
+    expect(getValue(item)).toEqual({set: "set"});
+
+    item = {
+      contents: {
+        get: "get",
+        set: "set"
+      }
+    };
+    expect(getValue(item)).toEqual({get: "get", set: "set"});
+  });
+});
```

Comment:
You might want to start breaking up the object-inspector tests file along describe sections.